src/common/mod.rs

```diff
@@ -24,6 +24,7 @@
 //! - [gpio_motor]
 //! - [adxl345]
 //! - [mpu6050]
+//! - [ina]
```

Comment:
The other sections look alpha sorted, so this section probably should be too, even though it wasn't before.